src/Symfony/Component/Validator/Resources/translations/validators.uz.xlf

```diff
@@ -4,11 +4,11 @@
         <body>
             <trans-unit id="1">
                 <source>This value should be false.</source>
-                <target>Qiymat noto'g'ri bo'lishi kerak.</target>
+                <target>Qiymat false bo'lishi kerak.</target>
```

Comment:
why this change? `yaroqsiz`?

Reply:
Technical words should not be translated if i am not mistaken.